.github/dependabot.yml

```
@@ -5,6 +5,9 @@ updates:
     schedule:
       interval: "weekly"
   - package-ecosystem: "github-actions"
-    directory: "/"
+    directories:
+      - "/"
+      - "/.github/util/initialize/"
+      - "/.github/util/sass-spec/"
```

Can we consolidate these into

```suggestion
      - "/.github/util/*/"
```

---

Change committed. I haven't tested this but it seems that wildcard is supported according to dependabot doc.